CoEpi/repo/AlertRepo.swift

```diff
@@ -13,4 +15,9 @@ class AlertRepoImpl: AlertRepo {
     func alerts() -> [Alert] {
         alertData
     }
+
+    func removeAlert(alert: Alert) {
```

Comment:
We should use rx in the repos too. So
- `alertData` becomes an observable (a subject, such that you can push data to it)
- In the view model you just have to map this observable to the outputs of the view model.
- When removing, you can either manipulate directly the value and push a new one or push a "delete event" that is combined with the alerts observable and pushes a new value to it. The later is relatively complicated. So it's ok if you implement the former.

Reply:
- I think we'd want it to still be a Relay right? Relays are `A Subject except without the ability to call onComplete or onError`

But will make this change

Reply:
Updated, not sure if I did this right. I just moved the `BehaviorRelay` into the alert repo and made it part of the interface, as is done for `Central` and `Peripheral`.

When you say mapping the observable, are you referring to making `Driver`s out of it? Or mapping it to a `AlertViewState` object from the domain `Alert` model?. Don't think I'm following the combining observables portion... I think it's my inexperience with Rx, I assume it's not just having the relay accept a new value, which pushes state to the driver?

Reply:
For now a Relay is fine. Though at some point this will come from an api or the local database and we'd need error handling (unless we model the error flow via a `Result` type. But it's probably better to stick to the default rx error handling).

What I mean with mapping is deriving new observables from this observable, which you're already doing in the view model.